.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- If this pull requests solves an issue, as an example, use `Closes #31`. #31 stands for the issue number -->
+
+## Fixes Issue
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
+
+## Changes proposed
+
+<!-- Here comes all the changes proposed through this PR -->
+
+<!-- Check all the boxes which are applicable to check the box correct follow the following conventions -->
+<!--
+[x] - Correct
+[X] - Correct
+-->
+
+## Check List (Check all the boxes which are applicable) <!-- Follow the above conventions to check the box -->
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
+
+## Screenshots
+
+<!-- Add all the screenshots which support your changes -->
```

Comment:
Remove 'all the' so it reads "Add screenshots which support your changes"

Reply:
```suggestion
<!-- Add screenshots which support your changes -->
```